src/javascripts/components/Result.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import User from "./User.js"
+import City from "./City.js"
+import Team from "./Team.js"
+
+import "../styles/Result.css"
+
+class Result extends Component {
```

Comment:
@forsaken1 

- user props, don't use state in container component, just props (you don't need to edit anything here)
- move component into .../containers/Results.js